doc/manual/parallel-computing.rst

```diff
@@ -562,6 +562,10 @@ arrays, a shared array is accessible only from those participating
 workers specified by the ``pids`` named argument (and the creating
 process too, if it is on the same host).
 
+Note: If you want to store a Type like object, it must be immutable to count
+as a [bitstype](http://docs.julialang.org/en/release-0.4/manual/types/). 
```

Comment:
The document is wrong here, it should be [`isbits`](http://julia.readthedocs.io/en/latest/stdlib/base/?highlight=isbits#Base.isbits) not `bitstype`. Also, I believe you are using the markdown syntax instead of rst cross reference and you are linking the 0.4 doc here.